code/modules/reagents/chemistry/reagents/impure_reagents/impure_medicine_reagents.dm

```diff
@@ -0,0 +1,521 @@
+//Reagents produced by metabolising/reacting fermichems inoptimally these specifically are for medicines
+//Inverse = Splitting
+//Invert = Whole conversion
+//Failed = End reaction below purity_min
+
+//START SUBTYPES
+
+//We don't want these to hide - they're helpful!
+/datum/reagent/impurity/healing
+	name = "Healing impure reagent"
+	description = "Not all impure reagents are bad! Sometimes you might want to specifically make these!"
+	chemical_flags = REAGENT_DONOTSPLIT
+	addiction_types = list(/datum/addiction/medicine = 3.5)
+	liver_damage = 0
+
+/datum/reagent/inverse/healing
+	name = "Healing inverse reagent"
+	description = "Not all impure reagents are bad! Sometimes you might want to specifically make these!"
+	chemical_flags = REAGENT_DONOTSPLIT
+	addiction_types = list(/datum/addiction/medicine = 3)
+	tox_damage = 0
+
+// END SUBTYPES
+
+////////////////////MEDICINES///////////////////////////
+
+//Catch all failed reaction for medicines - supposed to be non punishing
+/datum/reagent/impurity/healing/medicine_failure
+	name = "Insolvent medicinal precipitate"
+	description = "A viscous mess of various medicines. Will heal a damage type at random"
+	metabolization_rate = 1 * REM//This is fast
+	addiction_types = list(/datum/addiction/medicine = 7.5)
+	ph = 11
+
+//Random healing of the 4 main groups
+/datum/reagent/impurity/healing/medicine_failure/on_mob_life(mob/living/carbon/owner, delta_time, times_fired)
+	var/pick = pick("brute", "burn", "tox", "oxy")
+	switch(pick)
+		if("brute")
+			owner.adjustBruteLoss(-0.5)
+		if("burn")
+			owner.adjustFireLoss(-0.5)
+		if("tox")
+			owner.adjustToxLoss(-0.5)
+		if("oxy")
+			owner.adjustOxyLoss(-0.5)
+	..()
+
+// C2 medications
+// Helbital
+//Inverse:
+/datum/reagent/inverse/helgrasp
+	name = "Helgrasp"
+	description = "This rare and forbidden concoction is thought to bring you closer to the grasp of the Norse goddess Hel."
+	metabolization_rate = 1*REM //This is fast
+	tox_damage = 0.25
+	ph = 14
+	//Compensates for delta_time lag by spawning multiple hands at the end
+	var/lag_compensate = 0
+
+//Warns you about the impenting hands
+/datum/reagent/inverse/helgrasp/on_mob_add(mob/living/L, amount)
+	to_chat(L, "<span class='hierophant'>You hear laughter as malevolent hands apparate before you, eager to drag you down to hell...! Look out!</span>")
+	playsound(L.loc, 'sound/chemistry/ahaha.ogg', 80, TRUE, -1) //Very obvious tell so people can be ready
+	. = ..()
+
+//Sends hands after you for your hubris
+/datum/reagent/inverse/helgrasp/on_mob_life(mob/living/carbon/owner, delta_time, times_fired)
+	spawn_hands(owner)
+	lag_compensate += 1 - max(delta_time, 1)
+	. = ..()
+
+/datum/reagent/inverse/helgrasp/proc/spawn_hands(mob/living/carbon/owner)
+	if(!owner)//Unit test has a very strange interation where hands spawned will no longer have a target causing the beam to runtime
```

Comment:
This is caused by `/obj/projectile/pixel_move` deleting the projectile when the turf to move onto next doesn't exist, and then `/obj/projectile/curse_hand/Destroy` drawing a beam between where the projectile was fired and the turf that it would have moved to next.

Reply:
`/obj/projectile/curse_hand/Destroy` drawing a beam was already removed by me IIRC. Might want to double check if this is still possible.

Reply:
This should be fixed by #57271

Reply:
My bad, I should have noticed my local repository not having updated in the last 11 days.